openassessment/xblock/openassessmentblock.py

```diff
@@ -46,43 +46,36 @@
     "submission": {
         "name": "submission",
         "class_id": "openassessment__response",
-        "navigation_text": "Your response to this assignment",
         "title": "Your Response"
     },
     "student-training": {
         "name": "student-training",
         "class_id": "openassessment__student-training",
-        "navigation_text": "Learn to assess responses",
         "title": "Learn to Assess"
     },
     "peer-assessment": {
         "name": "peer-assessment",
         "class_id": "openassessment__peer-assessment",
-        "navigation_text": "Your assessment(s) of peer responses",
         "title": "Assess Peers' Responses"
     },
     "self-assessment": {
         "name": "self-assessment",
         "class_id": "openassessment__self-assessment",
-        "navigation_text": "Your assessment of your response",
         "title": "Assess Your Response"
     },
     "staff-assessment": {
         "name": "staff-assessment",
         "class_id": "openassessment__staff-assessment",
-        "navigation_text": "Staff assessment of your response",
-        "title": "Staff Assessment"
+        "title": "Staff Grade"
```

Comment:
@cahrens I think "Staff Grade" should have a colon after it like "Your Grade:" in line 74 below.